### ISSUE TITLE
dashboard: use folium for mapping sites

### ISSUE THREAD

**Issue by dacoex** [sandialabs/pecos#29]
See page example at:

- https://pypi.python.org/pypi/folium / section: Vincent/Vega Markers
- http://python-visualization.github.io/folium/quickstart.html#vincent-vega-markers
- http://bl.ocks.org/wrobstory/5609803

**Comment by kaklise** [comment]
This looks great.  Thanks for sharing the links.  I'll check it out and try to make a dashboard example using this capability.  It seems like marker could link to a single graphic or a HTML report.

**Comment by dacoex** [comment]
yes, just sharing. my personal use for pecos is as script only. Ex-post analysis / QC of recoded datasets

**Comment by dacoex** [comment]
For static maps, this one is helpful:
https://github.com/SciTools/cartopy
